deploy/kubernetes/dolphinscheduler/Chart.yaml

```diff
@@ -16,7 +16,7 @@
 #
 
 apiVersion: v2
-name: dolphinscheduler
+name: dolphinscheduler-helm
```

Comment:
For release managers, if you plan to release Helm Chart for old versions, you need to modify this